extensions/arc/src/ui/dashboards/postgres/postgresWorkerNodeParametersPage.ts

```diff
@@ -36,33 +36,42 @@ export class PostgresWorkerNodeParametersPage extends PostgresParametersPage {
 	}
 
 	protected async saveParameterEdits(engineSettings: string, session: azdataExt.AzdataSession): Promise<void> {
-		await this._azdataApi.azdata.arc.postgres.server.edit(
-			this._postgresModel.info.name,
-			{ engineSettings: engineSettings },
-			this._postgresModel.engineVersion,
-			this._postgresModel.controllerModel.azdataAdditionalEnvVars,
-			session);
-
+		try {
+			await this._azdataApi.azdata.arc.postgres.server.edit(
+				this._postgresModel.info.name,
+				{ engineSettings: engineSettings },
+				this._postgresModel.engineVersion,
+				this._postgresModel.controllerModel.azdataAdditionalEnvVars,
+				session);
+		} catch (error) {
```

Comment:
Remove the try/catch - it's not doing anything since if the awaited call throws it'll cause this function to throw as well